src/actions.js

```diff
@@ -84,6 +85,11 @@ export function fetchGroupHistory(params) {
   return graphql(payload, ACTION_TYPE.SEARCH_GROUP_HISTORY);
 }
 
+export function fetchBenefitPlanSchemaFields(params) {
```

Comment:
We have to think if this is a proper way to call to social protection gql endpoint to obtain schema.

Reply:
maybe we could add contribution or check if there is social protection installed - show additional fields. Otherwise limit exporting to basic permanent fields from database like firstname, lastname, dob